scripts/prereg/approve_draft_registrations.py

```diff
@@ -5,7 +5,7 @@
 import logging
 
 from website.app import init_app
-from website.models import DraftRegistration, Sanction, User
+from website.project.model import DraftRegistration, Sanction, User
```

Comment:
Why change this? To conform to style elsewhere? To avoid a circular import? ... ?